services/webinar/authenticate.test.js

```diff
@@ -0,0 +1,79 @@
+// @flow
+
+import { expect } from 'chai';
+import nock from 'nock';
+
+import { apiBase } from './utils';
+import ApiCredentialsManager from './authenticate';
+
+function expectHttpRequest(status) {
+  const reqheaders = {
+    accept: 'application/json',
+    'content-type': 'application/x-www-form-urlencoded',
+  };
+
+  const reqbody = 'grant_type=password&user_id=USER&password=PASSWORD&client_id=CLIENT';
+
+  const resbody = {
+    access_token: 'token', // eslint-disable-line camelcase
+    organizer_key: 'organizer', // eslint-disable-line camelcase
+    expires_in: '100', // eslint-disable-line camelcase
+  };
+
+  nock(apiBase, { reqheaders })
+    .post('/oauth/access_token', reqbody)
+    .reply(status, resbody);
+}
+
+describe('webinar-registration-service/authenticate', () => {
+  process.env.GOTOWEBINAR_USER_ID = 'USER';
+  process.env.GOTOWEBINAR_PASSWORD = 'PASSWORD';
+  process.env.GOTOWEBINAR_CLIENT_ID = 'CLIENT';
```

Comment:
potentially dumb question - do these stay as these values after you've run the test?

Reply:
discussed - pass them in as default variables